src/main/java/ru/practicum/shareit/item/ItemRepositoryInMemory.java

```diff
@@ -0,0 +1,71 @@
+package ru.practicum.shareit.item;
+
+import org.springframework.stereotype.Repository;
+import ru.practicum.shareit.exception.NotFoundException;
+import ru.practicum.shareit.item.model.Item;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Repository
+public class ItemRepositoryInMemory implements ItemRepository {
+    private final HashMap<Long, Item> itemMap = new HashMap<>();
```

Comment:
Лучше обозначать интерфейс, а уже инициализировать объект с определенной реализацией. Так программа становится более гибкой и не сломается, если нужно будет подменить реализацию на другую
```suggestion
    private final Map<Long, Item> itemMap = new HashMap<>();
```